gatsby-config.js

```diff
@@ -94,7 +95,74 @@ module.exports = {
               }
             `,
             output: "/rss.xml",
-            title: "Layer5"
+            title: "Layer5",
+          },
+        ],
+      },
+    },
+    {
+      resolve: "gatsby-plugin-feed-mdx",
```

Comment:
Did you check if adding the whole code under the first feed itself works or not?
Ultimately `feeds` is an array of objects, so it should work (I might be wrong though), but better to check so that we can avoid duplicate declarations.
P.S: If it works, please update the feed for `rss-contributors` similarly.

Reply:
@Nikhil-Ladha Yes it is working, you can check in the preview, and will do the feed for `rss-contributors`

Reply:
And why do the url for news page is `/company/news` and there is no page `/company`

Reply:
I don't see any change being done 🤔 
The feeds for which each of them is still defined separately.
Try merging all of them under the first declaration of the plugin like:
```
      resolve: "gatsby-plugin-feed-mdx",
      options: {
        query: `
          {
            site {
              siteMetadata {
                title
                description
                siteUrl
                site_url: siteUrl
              }
            }
          }
        `,
        feeds: [
          {
            <Feed for "blogs", "news">
          },
          {
          <Feed for rss-contributors>
          },
          {
          <Feed for news>
          }
        ],
      },
    },

Reply:
I am not sure if this will work, so check it out

Reply:
> And why do the url for news page is `/company/news` and there is no page `/company`

Because 
1. "news" covers "company" wide topics
2. SEO